ci/test.bash

```diff
@@ -31,16 +31,30 @@ fi
 
 $CROSS test --target $TARGET_TRIPLE
 $CROSS run --target $TARGET_TRIPLE --manifest-path systest/Cargo.toml
-echo === zlib-ng build ===
+
+echo '::group::=== zlib-ng build ==='
 $CROSS test --target $TARGET_TRIPLE --no-default-features --features zlib-ng
 $CROSS run --target $TARGET_TRIPLE --manifest-path systest/Cargo.toml --no-default-features --features zlib-ng
-echo === libz-ng-sys build ===
+echo '::endgroup::'
+
+# Note we skip compiling these targets on CI because the gcc version currently used in
+# cross for them is 5.4, ~8 years old at this point, hopefully it will be updated...sometime
+skip_triples=("x86_64-unknown-linux-gnu" "i686-unknown-linux-gnu" "aarch64-unknown-linux-gnu" "arm-unknown-linux-gnueabihf" "s390x-unknown-linux-gnu")
+if [[ -z $CI ]] || ! [[ ${skip_triples[@]} =~ "${TARGET_TRIPLE}" ]]; then
+    echo '::group::=== zlib-ng-no-cmake build ==='
```

Comment:
This test would still halt CI on failure, even though it should produce a warning. If CI halts when this breaks unrelated changes are also held up even though they shouldn't.
The intended workflow is that I CC you when I see such warning.